site/content/in-dev/unreleased/evolution.md

```diff
@@ -0,0 +1,62 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+title: Polaris Evolution
+type: docs
+weight: 1000
+---
+
+This page discusses what can be expected from Apache Polaris as the project evolves.
+
+## Polaris as a Catalog
+
+Polaris is primarily intended to be used as a Catalog of Tables and Views. As such, 
+it implements the Iceberg REST API, Polaris Management API and the Generic Tables API.
+
+Revisions of the Iceberg REST API are controlled by the [Apache Iceberg](https://iceberg.apache.org/)
+community. Polaris attempts to accurately implement this specification. However, there
+is no guarantee that Polaris releases always implement the latest version of the Iceberg
+REST Catalog API.
```

Comment:
```suggestion
community. 
```
Is it beneficial to mention an ad-hoc approach here? Does this "Polaris attempts to accurately implement this specification." imply Polaris isn't 100% compatible with IRC? I'd suggest to just remove it.

Reply:
> What value do users get from this message?

Avoiding false expectation that a Polaris release on Jun 16, 2025 (hypothetical example) supports Iceberg REST Catalog API as it is defined in Iceberg v. `1.9.1`.

Reply:
I'll rephrase the "attempts" part to carry a stronger commitment.

Reply:
updated